src/components/TxBuilderResult.js

```diff
@@ -1,78 +1,130 @@
+        finalResult = (
+          <div>
+            Network Passphrase:
+            <br />
+            {this.props.networkPassphrase}
+            <br />
+            Hash:
+            <br />
+            <EasySelect>{transactionBuild.hash}</EasySelect>
+            <br />
+            XDR:
+            <br />
+            <EasySelect>{transactionBuild.xdr}</EasySelect>
           </div>
-        signingInstructions = <p className="TransactionBuilderResult__instructions">
-          In order for the transaction to make it into the ledger, a transaction must be successfully
-          signed and submitted to the network. The laboratory provides
-          the <a href="#txsigner">Transaction Signer</a> to for signing a
-          transaction, and the <a href="#explorer?resource=transactions&endpoint=create">Post Transaction endpoint</a> for
-          submitting one to the network.
-        </p>;
-        signingLink = <a className="s-button"
-          href={txSignerLink(transactionBuild.xdr)}
-          onClick={scrollOnAnchorOpen}>Sign in Transaction Signer</a>
-        xdrLink = <a className="s-button"
-          href={xdrViewer(transactionBuild.xdr, 'TransactionEnvelope')}
-          onClick={scrollOnAnchorOpen}>View in XDR Viewer</a>
+        );
+        signingInstructions = (
+          <p className="TransactionBuilderResult__instructions">
+            In order for the transaction to make it into the ledger, a
+            transaction must be successfully signed and submitted to the
+            network. The laboratory provides the{" "}
+            <a href="#txsigner">Transaction Signer</a> to for signing a
+            transaction, and the{" "}
+            <a href="#explorer?resource=transactions&endpoint=create">
+              Post Transaction endpoint
+            </a>{" "}
+            for submitting one to the network.
+          </p>
+        );
+        signingLink = (
+          <a
+            className="s-button"
+            href={txSignerLink(transactionBuild.xdr)}
+            onClick={scrollOnAnchorOpen}
+          >
+            Sign in Transaction Signer
+          </a>
+        );
+        xdrLink = (
+          <a
+            className="s-button"
+            href={xdrViewer(transactionBuild.xdr, "TransactionEnvelope")}
+            onClick={scrollOnAnchorOpen}
+          >
+            View in XDR Viewer
+          </a>
+        );
       }
     }
 
-    let errorTitle = errorTitleText ? <h3 className="TransactionBuilderResult__error">{errorTitleText}</h3> : null
-    let successTitle = successTitleText ? <h3 className="TransactionBuilderResult__success">{successTitleText}</h3> : null
+    let errorTitle = errorTitleText ? (
+      <h3 className="TransactionBuilderResult__error">{errorTitleText}</h3>
+    ) : null;
+    let successTitle = successTitleText ? (
+      <h3 className="TransactionBuilderResult__success">{successTitleText}</h3>
+    ) : null;
+
+    const txBuilderHasErrors =
```

Comment:
`transactionBuild` is an object so it was returning true regardless whether it had errors or not.